navigator-menu/src/components/tips-calculator-components/left-portion/tip-container/TipContainer.jsx

```diff
@@ -0,0 +1,49 @@
+import {
+  Container,
+  Label,
+  PercentageBtns,
+  Button,
+  InputBtn,
+} from './TipContainer.styles';
+import { useContext } from 'react';
+import { TipsCalculatorContext } from '../../../../contexts/TipsCalculatorContext';
+
+function TipContainer() {
+  const { percentage } = useContext(TipsCalculatorContext);
+
+  const handleBtnChange = (e) => {
```

Comment:
You don't need this. You can just do `onClick={() => handleBtnChange(5)}` in the DOM.

The value of the input field can be taken by a ref, not raw HTML Elements.